docs/proposals/organizations.md

```diff
@@ -0,0 +1,118 @@
+
+ 4.  Can we punt on project transfering?
+
+     An org-admin shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
+
+
+
+## Organizations
+An Organization is a cluster scoped API object that references Users and has subresources for accessing and modifying Organization scoped OrgGroups.
+```
+// Organization contains the set of Users that are members of the Organization.
+type Organization struct {
+	kapi.TypeMeta
+	kapi.ObjectMeta
+
+	// AdminUsers is a list of Users that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminUsers util.StringSet
+	// AdminGroups is a list of Groups that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminGroups util.StringSet
+
+	// MemberUsers is a list of Users that are members of the Organization.  Only members may be assigned to OrgGroups.
+	MemberUsers util.StringSet
+	// MemberGroups is a list of Groups whose constituent users are considered members of the Organization.  Only members may be assigned to OrgGroups.
+	MemberGroups util.StringSet
+}
+```
+
+The Organization has a list of admins.  Admins have the power to:
+ 1.  Reserve quota for particular projects.
+ 2.  Accept ownership of a project.
+ 3.  Offer ownership of a project.
+ 4.  Remove (but not add) group members.
+ 5.  Create and modify organization groups.
+ 6.  Modify RoleBindings in projects the organization owns.  (This prevents accidentally orphaning a project.)
+
+The Organization has a list of members.  Only users from this list may be assigned to OrgGroups.  This prevents an overzealous organization admin from creating OrgGroups made up of non-members.
+
+Organizations will have an `organizations/members` subresource that flattens the list of members by resolving the constituent Users of the MemberGroups.
+
+
+## Quota allocation
+An Organization can be assigned quota by a cluster-admin.  The simplest way to model this is to have a cluster-scoped `ResourceQuota` (We can't call it that because `ResourceQuota` is a namespace-scope resource and it can't be in both scope.) that is linked to an Organization.  The existing quota admission controller would remain in place, but no limits would be set by default.  If an Organization admin decided that a particular project was starving the rest, he could assign hard limits to that Project's `ResourceQuota` to prevent it from taking more.  An additional admission controller would be responsible for doing the rollup across all projects in the Organization and preventing over-use of resources.
```

Comment:
It's a quota that applies to a label selector of namespaces.  It's Probably ResourceQuotaPolicy or ResourceQuotaController or similar.